src/listr/tasks/startNodeTaskFactory.js

```diff
@@ -26,7 +26,7 @@ function startNodeTaskFactory(dockerCompose) {
   function startNodeTask(
     config,
     {
-      isFullNode,
+      isMasternode,
```

Comment:
We don’t need to pass it as an argument since we can read it from config

Reply:
I guess the same applies to `isMinerEnabled` then right? Both miner and masternode are now read from config instead of passed as arguments.

Reply:
Defo! Nice catch!